scripts/data_collector/pit/test_pit.py

```diff
@@ -9,15 +11,22 @@ class TestPIT(unittest.TestCase):
     """
     NOTE!!!!!!
     The assert of this test assumes that users follows the cmd below and only download 2 stock.
-    `python collector.py download_data --source_dir ./csv_pit --start 2000-01-01 --end 2020-01-01 --interval quarterly --symbol_flt_regx "^(600519|000725).*"`
+    1. `python scripts/get_data.py qlib_data --target_dir ~/.qlib/qlib_data/cn_data --region cn`
+    2. `python scripts/data_collector/pit/collector.py download_data --source_dir ~/.qlib/stock_data/source/pit --start 2000-01-01 --end 2020-01-01 --interval quarterly --symbol_regex "^(600519|000725).*"`
+    3. `python scripts/data_collector/pit/collector.py normalize_data --interval quarterly --source_dir ~/.qlib/stock_data/source/pit --normalize_dir ~/.qlib/stock_data/source/pit_normalized`
+    4. `python scripts/dump_pit.py dump --csv_path ~/.qlib/stock_data/source/pit_normalized --qlib_dir ~/.qlib/qlib_data/cn_data --interval quarterly`
     """
 
     def setUp(self):
         # qlib.init(kernels=1)  # NOTE: set kernel to 1 to make it debug easier
-        qlib.init()  # NOTE: set kernel to 1 to make it debug easier
+        qlib.init(provider_uri="~/.qlib/qlib_data/cn_data")
```

Comment:
`provider_uri="~/.qlib/qlib_data/cn_data"` is not necessary.  It's the default value
![image](https://user-images.githubusercontent.com/465606/158962533-5fd384fe-9e59-4ae3-82b1-a5163417a4e3.png)